src/main/java/com/hurko/model/Role.java

```diff
@@ -0,0 +1,6 @@
+package com.hurko.model;
+
+public enum Role {
```

Comment:
@dezuz What if we will make Role as an entity with RoleName enum?
```suggestion
public enum Role {
long id;
RoleName roleName;


enum RoleMName {USER, ADMIN}
```